### ISSUE TITLE
The -j flag default is missing

### ISSUE THREAD

**Issue by fancsali** [skonfig/skonfig#56]
Running
```
skonfig -j my.host.name
```
will show the usage message instead of defaulting to the numer of CPU's as said usage help would suggest:

```
$ skonfig -j my.host.name
usage: skonfig [-h] [-V] [-d] [-i path] [-j jobs] [-n] [-v] [host]

positional arguments:
host        host to configure

optional arguments:
-h, --help  show this help message and exit
-V          print version
-d          print dumped hosts, -d <host> = print dump
-i path     initial manifest or '-' to read from stdin
-j jobs     maximum number of jobs (defaults to host CPU count)
-n          dry-run, do not execute generated code
-v          -v = VERBOSE, -vv = DEBUG, -vvv = TRACE
```

**Comment by 4nd3r** [comment]
`skonfig` will always default to host CPU count. You have to use `-j` only if you want to change the default.

See discussion between me and @sideeffect42 in #54.

**Comment by fancsali** [comment]
I think, there's a bit of a mix-up; at least in my mind. Based on that comment, I assume (maybe wrongly), that the usual practice is in place:

* `skonfig my.host.name` -> 1 single process
* `skonfig -j my.host.name` -> number of CPUs processes
* `skonfig -j x my.host.name` -> x number of processes

But I sense now, it's **always** number of CPUs and never single-threaded, unless one explicitly specify `-j 1`; and then one can differ and go higher if `-j` is specified on the command line.

In other words, is there only two scenarios? Like below:
* `skonfig my.host.name` -> number of CPUs processes
* `skonfig -j x my.host.name` -> x number of processes

**Comment by 4nd3r** [comment]
> In other words, is there only two scenarios?

Yes. I fail to see why we don't want this behavior by default. What am I missing? Sure, this is how `cdist` behaves. Probably in order to surprise users, but I think we can allow this radical change :smile: 

I see this can become an issue if you run `skonfig` on host with too many cores and too many parallel connections to low power target host might have opposite effect. For example with RPis. My main desktop machine have 6 cores and RPi Zero W barely survives this. But this is my edge case and I just use `-j 2` then.
